luda/editor/client/src/render.ts

```diff
@@ -36,14 +36,10 @@ type State = {
 
 export function render(state: State): RenderingTree {
   const { editingSequenceTitle } = state.sequenceListViewState;
-  if (editingSequenceTitle) {
-    saver.autoSave(
-      `/sequence/${editingSequenceTitle}.json`,
-      state.timelineState.tracks,
-    );
-  }
 
-  if (!editingSequenceTitle) {
+  const isEditingSequence = editingSequenceTitle !== undefined;
```

Comment:
I describe this situation with variable name

Reply:
Let's use variable name as comment, to let code-reader know what kind of condition you want to handle.